.sbtrc

```diff
@@ -0,0 +1,5 @@
+# Test run control
+alias test=; testOnly -l IntegrationTest
+alias testDB=; testOnly -t IntegrationTest
+alias testIT=; testOnly -- -t IntegrationTest
```

Comment:
Could you please describe these switches? I tried to find their meaning but failed. 😞

Reply:
A new section was added to the README.md file. See commit 5c62f864b4d198b21e784c5eff24a6af5bc60e45.

Reply:
I didn't mean such a detailed description in `README.md ` (has the disadvantage of duplicating code) 😉 , although the description of aliases is good to have.

But I still don't understand the difference between `-t IntegrationTest` and `-- -t IntegrationTest`

Reply:
See the current solution - I refactored the solution. See README.md file.